src/lib/jsonParseV2/parseSubscription.cpp

```diff
@@ -100,6 +100,25 @@ std::string parseSubscription(ConnectionInfo* ciP, ParseData* parseDataP, JsonDe
     return oe.render(ciP, "");
   }
 
+  // Description field
+  destination->descriptionProvided = false;
+  if (document.HasMember("description"))
+  {
+    const Value& description      = document["description"];
+    std::string descriptionString = description.GetString();
```

Comment:
description doesn't seem to be used after this.
These two lines could be one (and not too long) line.

It's a matter of taste.

NTC